datafusion/sqllogictest/test_files/array.slt

```diff
@@ -1978,11 +1978,11 @@ select array_slice(arrow_cast(make_array(1, 2, 3, 4, 5), 'LargeList(Int64)'), 0,
 ----
 [1, 2, 3, 4, 5] [h, e, l, l, o]
 
-# TODO: Enable once arrow_cast supports ListView types.
+# TODO: Enable once array_slice supports LargeListView types.
 # Expected output (once supported):
 # ----
 # [1, 2, 3, 4, 5] [h, e, l, l, o]
-query error DataFusion error: Execution error: Unsupported type 'ListView\(Int64\)'. Must be a supported arrow type name such as 'Int32' or 'Timestamp\(ns\)'. Error unknown token: ListView
+query error Failed to coerce arguments to satisfy a call to 'array_slice' function:
```

Comment:
Might not be related to this PR but now I don't know what arguments are the problematic ones

before I could see it's `ListView` but now I have no idea what arguments is invalid and what their type that is not supported

Reply:
The actual error is like this:
```
	DataFusion error: Error during planning: Failed to coerce arguments to satisfy a call to 'array_slice' function: coercion from ListView(Int64), Int64, Int64 to the signature OneOf([ArraySignature(Array { arguments: [Array, Index, Index], array_coercion: Some(FixedSizedListToList) }), ArraySignature(Array { arguments: [Array, Index, Index, Index], array_coercion: Some(FixedSizedListToList) })]) failed No function matches the given name and argument types 'array_slice(ListView(Int64), Int64, Int64)'. You might need to add explicit type casts.
```

Something about that was causing issues with the slt error matching logic

This error is pretty terrible, so I will file a separate ticket to track improving it